src/values.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+#define FOCUSED_TALL_CELL_HEIGHT   60
```

Comment:
I believe these relevant values are also exported now (I realize this happened after you opened the PR, but we should update our example accordingly)